sdk/eventhub/Azure.Messaging.EventHubs.Shared/src/Testing/EventHubScope.cs

```diff
@@ -47,12 +55,15 @@ public sealed class EventHubScope : IAsyncDisposable
         ///
         /// <param name="eventHubHame">The name of the Event Hub that was created.</param>
         /// <param name="consumerGroups">The set of consumer groups associated with the Event Hub; the default consumer group is not included, as it is implicitly created.</param>
+        /// <param name="wasEventHubCreated">Sets whether the Event Hubs was created or read from environment variables.</param>
```

Comment:
(extreme) nit:  `Event Hubs` should be `Event Hub` here.  Definitely not a blocker.